packages/plugins/typescript/resolvers/tests/ts-resolvers.spec.ts

```diff
@@ -1426,10 +1426,38 @@ export type ResolverFn<TResult, TParent, TContext, TArgs> = (
       union CCCUnion = CCCFoo | CCCBar
     `);
 
-    const tsContent = (await tsPlugin(testSchema, [], {}, { outputFile: 'graphql.ts' })) as Types.ComplexPluginOutput;
+    const tsContent = await tsPlugin(testSchema, [], {}, { outputFile: 'graphql.ts' });
```

Comment:
`tsContent` should be typed as `Types.ComplexPluginOutput` so we don't need to do `as Types.ComplexPluginOutput`.  

Happy to put this back if it was here for some particular reasons